crates/bevy_ecs/src/component.rs

```diff
@@ -84,6 +84,21 @@ use std::{
 /// [`Table`]: crate::storage::Table
 /// [`SparseSet`]: crate::storage::SparseSet
 ///
+/// # Disabling Change Detection
```

Comment:
These docs need to be duplicated for the Resource trait.

Reply:
Added the same changes to Resources.